admin-dev/themes/default/js/bundle/product/product-combinations.js

```diff
@@ -21,6 +21,10 @@ var combinations = (function() {
           url: elem.attr('href'),
           beforeSend: function() {
             elem.attr('disabled', 'disabled');
+            $('#create-combinations').attr('disabled', 'disabled');
+            $('#apply-on-combinations').attr('disabled', 'disabled');
+            $('#submit', $('#form')).attr('disabled', 'disabled');
```

Comment:
I don't think you need context here since you're querying by ID (there should only be one element on the whole DOM with that ID)